src/applications/pensions/config/chapters/03-health-and-employment-information/helpers.js

```diff
@@ -265,3 +265,51 @@ export const generateEmployersSchemas = ({
     }),
   };
 };
+
+export function MedicalConditionDescription() {
+  return (
+    <div>
+      <p>
+        We need to know about any medical conditions that prevent you from
+        working.
+      </p>
+      <p>
+        A medical condition is an illness or injury that affects your mind or
+        body. It doesn't have to be service connected.
+      </p>
+      <va-additional-info
+        trigger="How we define a medical condition that prevents you from working"
+        uswds
+      >
+        <p>
+          If your medical condition prevents you from working, both of these
+          must be true:
+        </p>
+        <ul>
+          <li>
+            Your medical condition is reasonably certain to continue throughout
+            your lifetime, and
+          </li>
```

Comment:
I believe the bolded 'and' is intentional in this design per the design system guidelines for [bulleted lists](https://design.va.gov/content-style-guide/bulleted-lists#considerations).
```suggestion
          <li>
            Your medical condition is reasonably certain to continue throughout
            your lifetime, <strong>and</strong>
          </li>
```